### PR TITLE
Create two sections of docs

### PR DESCRIPTION
**Pull request by WhyFenceCode**
This commit adds me geolock branch in to the Iris main branch, adding tabs and tab refs to the main line of the Iris docs. While this may be pre mature for f-stop, this needs doing before any f-stop structuring can begin if we want to retain auto merge on future pr's. Therefore, I am merging this now before we begin to plan the new structure. This update only adds one tab, but has support for far more that we can add in the near future. This also changes all urls to work again.

### CODE REVIEW

File: src/content/docs/index.mdx [hunk 1]
```diff
@@ -13,11 +13,11 @@ hero:
     file: ../../assets/iris.webp
   actions:
     - text: Get Started
-      link: /guides/your-first-shader/0_intro/
+      link: /current/current/guides/your-first-shader/0_intro/
```
Comment (ninjamike1211) [inline src/content/docs/index.mdx:16]:
```suggestion
      link: /current/guides/your-first-shader/0_intro/
```

File: src/content/docs/index.mdx [hunk 2]
```diff
@@ -13,11 +13,11 @@ hero:
...
       icon: right-arrow
       variant: primary
     - text: Reference
-      link: /reference/overview/
+      link: /current/current/reference/overview/
```
Comment (ninjamike1211) [inline src/content/docs/index.mdx:20]:
```suggestion
      link: /current/reference/overview/
```

**Comment by jbritain** [comment]
Looks good to me, I'm not a huge fan of the styling of the tabs and I think we need a better name than 'current' but am happy to merge for the moment.

**Review by jbritain — APPROVED** [review]

**Comment by WhyFenceCode** [comment]
sounds good, i dont love the name either but what else would we call it?

**Review by ninjamike1211 — APPROVED** [review]
Looks good, we'll work on finalizing the appearance later but it's good to get the structure merged now.

**Pull request merged**